content/docs/rfcs/37/README.md

```diff
@@ -0,0 +1,151 @@
+
+## The $N11M$ session management mechanism
+
+In a $N11M$ setting, each party's device shares the same Noise session information used to encrypt and decrypt messages exchanged with the other party.
+
+![](./N11M.png)
+
+More precisely, once the first Noise session between any of Alice's and Bob's device is instantiated, 
+its session information is securely propagated to all other devices, 
+which then become able to send and receive new messages on the content topic associated to such session.
+
+The most updated session information between Alice and Bob is propagated in encrypted form to other devices, 
+using previously instantiated Noise sessions. 
+The propagated session information corresponds to the latest session information stored on the device currently communicating with (any of the devices of) the other party.
+
+We note that sessions information is propagated only among devices belonging to the same party and not with other party's devices. 
+Hence, Alice has no knowledge on the number of devices Bob is using and vice versa. 
+
+When any device marks a Noise session between Alice and Bob as stale, 
+all (updated) devices will implicitly mark such session as stale.
+
+In case a Noise session between two devices belonging to the same party is marked as stale, 
+such two devices stop to reciprocally propagate any information regarding Noise sessions instantiated with other parties.
+
+As regards security, an attacker that compromises an encrypted message propagating session information,
+might be able to compromise one or multiple messages exchanged within the session such information refers to. 
+This can be mitigated by adopting techniques similar to the the ones proposed in [35/WAKU2-NOISE](https://rfc.vac.dev/spec/35/#after-handshake), 
+where encryption keys are changed every time a new message is exchanged.
+
+This session management mechanism is loosely based on the paper ["Multi-Device for Signal"](https://eprint.iacr.org/2019/1363.pdf).
+
+## The $NM$ session management mechanism
+
+In a $NM$ setting, we require all of $N$ Alice's devices to have an active Noise session with each of Bob's $M$ devices, 
+for a total of $NM$ concurrently active Noise sessions between Alice and Bob.
+
+![](./NM.png)
+
+A message is sent from the currently-in-use sender's device to all recipent's devices, 
+by properly encrypting and sending it to the content topics of each corresponding active Noise session. 
+
+We note that this allows the recipient to receive a message on all his devices simultaneously. 
+However, on the sender side, only the device which effectively sent the message will know its full content.
+
+If it is required for sent messages to be available on all sender's devices, 
+each pair of sender's devices SHOULD have an active Noise session used for syncing purposes: 
+this sums up to a total of $N-1$ and $M-1$ extra Noise sessions instantiated on each Alice's and Bob's device, respectively.
```

Comment:
We cannot guarantee which device is active now, hence I'd say there should be a session for each pair of devices of the same person, like a full graph, thus N*(N-1)/2 sessions.

Reply:
Indeed, this is what is written: $N-1$ extra session per device, means that every device has a session with all the remaining $N-1$ devices, hence a total of $N(N-1)/2$ unique sessions.